_includes/ios/objects.md

```diff
@@ -344,43 +352,38 @@ gameScore.saveInBackground()
 ```
 </div>
 
-Note that it is not currently possible to atomically add and remove items from an array in the same save.
-    You will have to call `save` in between every different kind of array operation.
+Note that it is not currently possible to atomically add and remove items from an array in the same save using. You will have to call `save` in between every different kind of array operation.
 
 ## Deleting Objects
 
-To delete an object from the cloud:
-
-<div class="language-toggle" markdown="1">
-```objective_c
-[gameScore deleteInBackground];
-```
-```swift
-gameScore.deleteInBackground()
-```
-</div>
+There are a few ways to delete a  `PFObject`.  For basic asynchronous deletion of a single object call the objects `deleteInBackground` function. If you prefer to recieve a callback you can use the `deleteInBackgroundWithBlock:` or `deleteInBackgroundWithTarget:selector:` methods. If you want to block the calling thread, you can use the `delete` method. Lastly,  `deleteEventually`  network consious solution that deletes when possible but does not guarantee a timeframe for the tasks completion.
 
-If you want to run a callback when the delete is confirmed, you can use the `deleteInBackgroundWithBlock:` or `deleteInBackgroundWithTarget:selector:` methods. If you want to block the calling thread, you can use the `delete` method.
+For deleting multiple objects use the `PFObject` static function `deleteAllInBackground` to delete an array of objects asynchronous. The same can be done while blocking the calling thread using `deleteAll`. Lastly, to recieve a callback after deleting objects asyncronously use `deleteAllInBackground:block:` as demonstrated below.
```

Comment:
```suggestion
For deleting multiple objects use the `PFObject` static function `deleteAllInBackground` to delete an array of objects asynchronously. The same can be done while blocking the calling thread using `deleteAll`. Lastly, to recieve a callback after deleting objects asyncronously use `deleteAllInBackground:block:` as demonstrated below.
```

Reply:
God I feel so stupid for missing stupid stuff like spelling. I guess after about 5+ hours at it last night I started getting a bit sloppy. Theres just so much that needs to be done.